src/db/tripledb.plt

```diff
@@ -110,5 +110,19 @@ test('tripledb tell triple with Unit'):-
     assert_true(tripledb_tell(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter'))),
     assert_true(tripledb_ask(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter'))).
 
+% test for special characters in iri: @*~!#?
+test('tripledb_tell_special_characters_@*~!#?'):-
+    tripledb_tell(test_datatype:'normal_user_test_new', test_datatype:'last@*~!#?_name', 'umlaut').
```

Comment:
so did the tell work? why not check if you can retrieve it via ask?

Reply:
yes it works and ask is also added in new commit.